src/actions.js

```diff
@@ -341,7 +384,9 @@ export function getActionDefinitions(self) {
 			name: 'Lens - Follow Focus',
 			options: optSetIncDecStep('Focus setting', 0x555, 0x0, 0xaaa, 10),
 			callback: async (action) => {
-				await self.getPTZ('AXF' + cmdValue(action, 0x555, 0x0, 0xaaa, action.options.step, 3, self.data.focusPosition))
+				const parsedAction = await parseSetIncDecVariables(action, self, 0x0, 0xaaa, 10)
+				if (parsedAction === undefined) return
```

Comment:
Did I miss this action? Might need to add it to the upgrade script / presets.

Reply:
Yep. Fixed it.